.github/workflows/build-and-publish.yaml

```diff
@@ -92,6 +92,7 @@ jobs:
                 id: build-and-push
                 with:
                     file: Dockerfile
+                    platforms: linux/arm64/v8,linux/arm/v7,linux/amd64
```

Comment:
The [base image](https://hub.docker.com/layers/library/eclipse-temurin/21.0.6_7-jre-alpine/images/sha256-f184bb601f9e6068dd0a92738764d1ff447ab68c15ddbf8c303c5c29de9a1df8) we use for this doesn't support arm/v7.